src/frontend/src/utils/iiConnection.ts

```diff
@@ -719,7 +718,7 @@ export const creationOptions = (
       name: "Internet Identity Service",
     },
     user: {
-      id: tweetnacl.randomBytes(16),
+      id: window.crypto.getRandomValues(new Uint8Array(16)),
```

Comment:
Wasn't the underlying implementation of tweetnacl adding some complexity to `new Unit8Array(16)` or it's enough?

https://github.com/dchest/tweetnacl-js/blob/fecde6ecf0eb81e31d54ca0509531ab1b825f490/nacl.js#L1160

Reply:
This does not add complexity. It just invokes the browser API in batches of 65536 bytes because that is the limit of the browser API.
But we don't need that here.

Reply:
Gotcha, thanks for the feedback.